com.opera.Opera.yaml

```diff
@@ -0,0 +1,120 @@
+app-id: com.opera.Opera
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.chromium.Chromium.BaseApp
+base-version: '22.08'
+command: opera
+separate-locales: false
+finish-args:
+  - --require-version=1.8.2
+  - --device=all
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --share=ipc
+  - --share=network
+  - --socket=cups
+  - --socket=pcsc # FIDO2
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --system-talk-name=org.freedesktop.UPower
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.freedesktop.FileManager1
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.kde.kwalletd5
+  - --talk-name=org.gnome.SessionManager
+  - --system-talk-name=org.freedesktop.Avahi
+  - --own-name=org.mpris.MediaPlayer2.chromium.*
+  - --filesystem=/run/.heim_org.h5l.kcm-socket
+  - --persist=.pki
+
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    add-ld-path: .
+    autodelete: false
+    autodownload: true
+    directory: lib/ffmpeg
+    version: '22.08'
+
+  org.chromium.Chromium.Codecs:
+    directory: lib/nonfree-codecs
+    add-ld-path: lib
+    bundle: true
+    autodelete: true
```

Comment:
Judging from https://github.com/archlinux/svntogit-community/blob/packages/opera-ffmpeg-codecs/trunk/PKGBUILD, you need to mount it at `lib/opera/lib_extra/`.